content/chainguard/chainguard-images/getting-started/getting-started-python.md

```diff
@@ -160,7 +160,7 @@ To begin, create a directory for your app. You can use any meaningful name and p
 mkdir ~/inky/ && cd $_
 ```
 
-We'll first write out the requirements for our app in the `requirements.txt` file. We'll download the most recent version of Python [setuptools](https://pypi.org/project/setuptools/) at the time of writing, and also install [climage](https://pypi.org/project/climage/).
+We'll first write out the requirements for our app in a new file, for example we named our file `requirements.txt`. We'll download the most recent version of Python [setuptools](https://pypi.org/project/setuptools/) at the time of writing, and also install [climage](https://pypi.org/project/climage/).
```

Comment:
Should we move L170 (well, modify it to fit here with `nano requirements.txt` so that they know how to edit the file?